packages/config/src/config/imagery.ts

```diff
@@ -21,4 +21,23 @@ export interface ConfigImagery extends BaseConfig {
 
   /** list of file basenames and their bounding box */
   files: NamedBounds[];
+
+  /** Separate overview cache */
+  overviews?: ConfigImageryOverview;
+}
+
+export interface ConfigImageryOverview {
+  /** Path to overview archive */
+  path: string;
```

Comment:
This should be relative path?

Reply:
Can we update the comment to say this is a relative path from the imagery.uri?

Reply:
interesting thought, maybe we should check if it starts with `./` then join it or allow it to be absolute? `s3://foo/bar/overviews`

Reply:
That would be better. I would like to have the option to put overview in different location.